src/borg/testsuite/archive.py

```diff
@@ -34,23 +34,22 @@ def test_stats_basic(stats):
     assert stats.usize == 20
 
 
-def tests_stats_progress(stats, monkeypatch, columns=80):
+@pytest.mark.parametrize(
+    "item_path, update_size, expected_output",
+    [
+        ("", 0, "20 B O 20 B U 1 N "),  # test unchanged 'stats' fixture
+        ("foo", 10**3, "1.02 kB O 20 B U 1 N foo"),  # test updated original size and set item path
+        ("foo" * 40, 10**3, "1.02 kB O 20 B U 1 N foofoofoofoofoofoofoofoofo...foofoofoofoofoofoofoofoofoofoo"),
+    ],  # test long item path which exceeds 80 characters
```

Comment:
the comment belongs to line 42. so either put it there (if it does not exceed max line length) or above that line.